src/module/rolls/preroll-dialog.ts

```diff
@@ -119,6 +122,7 @@ export class IronswornPrerollDialog extends Dialog<
 
   static get defaultOptions() {
     return mergeObject(super.defaultOptions, {
+      // FIXME: 'dialog' class may be unnecessary since the parent object will reliably have [role=dialog] selector
```

Comment:
This is a carrying forward of the parent class's default options. Here's what it looks like in foundry.js:

```js
class Dialog extends Application {
  [...]
  static get defaultOptions() {
    return foundry.utils.mergeObject(super.defaultOptions, {
      template: "templates/hud/dialog.html",
      classes: ["dialog"],
      width: 400,
      jQuery: true,
    });
  }
```

`mergeObject` doesn't extend arrays, so we have to include the original value here.